app/src/end2endTest/java/gov/va/vro/end2end/VroV2Tests.java

```diff
@@ -134,6 +152,28 @@ private void testAutomatedClaimFullPositive(String collectionId, boolean expecte
     String expectedMessage = String.format("Received Claim for collection Id %s.", collectionId);
     assertEquals(expectedMessage, masResponse.getMessage());
 
+    boolean successOrdering = false;
+    if (expectedExamOrder != null) {
```

Comment:
I feel like this piece can be pulled out into a reusable private helper method... but don't feel strongly about it enough to make a formal change request.